Engine/src/Engine/Entity.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include "entt.h"
+#include "Scene.h"
+
+namespace Engine
+{
+	// Wrapper class for ENTT entity
+	// Credit to the following video for most of the code: https://youtu.be/8LbVpkEqKuY
+	class Entity
+	{
+	private:
+		entt::entity m_entityHandle = entt::null;
+		Scene* m_scene = nullptr;
+	public:
+		Entity() = default;
+		Entity(entt::entity entity_handle, Scene* scene);
```

Comment:
same thing as earlier, use entityHandle insead of entity_handle

Reply:
done